app-emulation/cap32_libretro/cap32_libretro-4.2.0_20171109.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A port of Caprice32, an Amstrad CPC emulator, to the libretro API"
+DESCRIPTION="Caprice32 is a software emulator of the Amstrad CPC 8bit home \
+computer series. The emulator imitates the CPC464, CPC664, and CPC6128 models."
+HOMEPAGE="https://sourceforge.net/projects/caprice32/"
+COPYRIGHT="1997-2018, Ulrich Doewich, dantoine the libretro team"
```

Comment:
Do we get rid of two commas?
```diff
-computer series. The emulator imitates the CPC464, CPC664, and CPC6128 models."
+computer series. The emulator imitates the CPC464, CPC664 and CPC6128 models."
 HOMEPAGE="https://sourceforge.net/projects/caprice32/"
-COPYRIGHT="1997-2018, Ulrich Doewich, dantoine the libretro team"
+COPYRIGHT="1997-2018 Ulrich Doewich, dantoine the libretro team"
```
I'm not 100% sure about the first one, i.e. before the "and".
BTW, in `COPYRIGHT`, is "dantoine the libretro team" OK?

Reply:
dantoine is a separate copyright holder
I fixed that